nexus/src/external_api/http_entrypoints.rs

```diff
@@ -1952,7 +2106,9 @@ async fn vpc_routers_get_router(
 }
 
 /**
- * Create a VPC Router
+ * Create a router
+ *
+ * Creates a new router in a VPC.
  */
```

Comment:
This should specify VPC Custom Router. There are two types of VPC routers: System and Custom. The System Router is implicitly created as part of the VPC and can neither be created nor destroyed explicitly. As new VPC Subnets are added, routes are added to the System Router implicitly. However, in order to refine routing policy a Custom Router can be created and associated with a VPC Subnet. Only one Custom Router per VPC Subnet. This is discussed in RFD 63 §2.3.

So I believe this API comment should specify VPC Custom Router to make it clear what it applies to. The same goes for the other router APIs.

Reply:
will elaborate

Reply:
> Only one Custom Router per VPC Subnet. This is discussed in RFD 63 §2.3.
one question if there is only one vpc router per subnet should this route be instead under /subnets/{subnet_name} for the endpoint path? to declare that subnet as the parent or it is one per VPC.